commit 6e8966a83001e9996180efa95a7fb46ab6784a8c
Author: q66 <q66@chimera-linux.org>
Date:   Tue Dec 5 01:39:54 2023 +0100

    disable vec_xl implementation for non-vsx case
    
    It seems clang altivec.h actually defines it unconditionally.

diff --git a/libswscale/ppc/yuv2rgb_altivec.c b/libswscale/ppc/yuv2rgb_altivec.c
index 5e1033a..23b4834 100644
--- a/libswscale/ppc/yuv2rgb_altivec.c
+++ b/libswscale/ppc/yuv2rgb_altivec.c
@@ -284,7 +284,7 @@ static inline void cvtyuvtoRGB(SwsContext *c, vector signed short Y,
  * ------------------------------------------------------------------------------
  */
 
-#if !HAVE_VSX
+#if 0
 static inline vector unsigned char vec_xl(signed long long offset, const ubyte *addr)
 {
     const vector unsigned char *v_addr = (const vector unsigned char *) (addr + offset);
